app/upgrades/v15/upgrade_test.go

```diff
@@ -201,7 +201,8 @@ func (suite *UpgradeTestSuite) TestSetICQParams() {
 	v15.SetICQParams(suite.Ctx, suite.App.ICQKeeper)
 
 	suite.Require().True(suite.App.ICQKeeper.IsHostEnabled(suite.Ctx))
-	suite.Require().Len(suite.App.ICQKeeper.GetAllowQueries(suite.Ctx), 65)
+	// TODO: commented out for historical reasons since v15 upgrade is now over.
```

Comment:
```suggestion
	// commented out for historical reasons since v15 upgrade is now over.
```
Removing the TODO to make searching through the repo for actionable items easier